reservation/test_appointment_model.py

```diff
@@ -0,0 +1,73 @@
+from reservation.models import Appointment, TypeOfJob
+from datetime import timedelta
+from django.utils import timezone
+import pytest
+
+current_datetime = timezone.now()
+START_APPOINTMENT = (current_datetime + timedelta(days=5)).replace(hour=12, minute=0, second=0, microsecond=0)
+END_APPOINTMENT = (current_datetime + timedelta(days=5)).replace(hour=13, minute=0, second=0, microsecond=0)
+SUMMARY = ""
+
+
+@pytest.fixture
+def persisted_appointment(appointment):
+    appointment.typeOfJob_id.save()
+    appointment.professional_id.save()
+    appointment.client_id.save()
+    appointment.save()
+    return appointment
+
+
+@pytest.fixture
+def persisted_appointment_pool(persisted_appointment, typeOfJob, professional, client2):
+    appointment = Appointment(professional_id=professional,
```

Comment:
See the problem here 
That you call one fixture "persisted_appointment"
You have one appointment
Then in line n23 you are creating another appointment,

Please use fixtures or factory, consult with team members already implemented it

Reply:
Done